src/app.js

```diff
@@ -660,7 +660,10 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
     editor: editor.event,
     staticAnalysis: staticanalysis.event
   }
-  self._components.righthandpanel = new RighthandPanel(rhpAPI, rhpEvents, {})
+  var rhpOpts = { api: {}, events: {} }
```

Comment:
we just want to remove the container argument in this PR. 
could we keep having `(api, event, opt)` ?